test/config.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from "chai";
+import { shallow } from "enzyme";
+import * as React from "react";
```

Comment:
you dont need react, enzyme here. Actually, linter should catch it.... but it didn't do you can you try investigating it why it happened? and make it work?